src/decode.js

```diff
@@ -116,11 +116,13 @@ function transformScalars(context, value) {
 }
 
 function recordTypeInformation(context, value) {
+  const selectionSet = context.selection.selectionSet;
```

Comment:
Just for reference, you can also [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) the `context.selection.selectionSet` like this `const {typeBundle, typeSchema} = context.selection.selectionSet;` This way you can directly use the `typeBundle` and `typeSchema` constants afterwards.